models/course.py

```diff
@@ -0,0 +1,9 @@
+from odoo import models, fields
```

Comment:
Please rename the file. See [guideline](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#15file-naming) for file naming please.